bottles/backend/managers/manager.py

```diff
@@ -303,7 +301,6 @@ def organize_dependencies(self):
     @RunAsync.run_async
     def organize_installers(self):
         """Organizes installers into supported_installers."""
-        RepoStatus.repo_start_operation("installers.organizing")
```

Comment:
Just cuirous, why removing these organizing lines?

Reply:
`start_operation` is not needed, only `wait` and `end`. When events are created, they are not set by default so it doesn't matter if we start or not the operation.